src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsList.types.ts

```diff
@@ -0,0 +1,20 @@
+import { IStyle, IStyleFunctionOrObject, Theme } from '@fluentui/react';
+
+export interface IConditionsListProps {
+    styles?: IStyleFunctionOrObject<
+        IConditionsListStylesProps,
+        IConditionsListStyles
+    >;
+}
+
+export interface IConditionsListStyles {
+    container: IStyle;
+    subComponentStyles?: IConditionsListSubcomponentStyles;
+}
+
+export interface IConditionsListStylesProps {
+    theme: Theme;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
just as a note, will this also be a part of the next PR (adding styles for subcomponents for the conditions list)?